### PR TITLE
man: explain max CPU load on cgtop

### PR DESCRIPTION
**Pull request by utezduyar**
Solves https://github.com/systemd/systemd/issues/159

### CODE REVIEW

File: man/systemd-cgtop.xml [hunk 1]
```diff
@@ -85,6 +85,11 @@
     <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>
     for details.</para>
 
+    <pare>The CPU load value can be between 0 and 100 x number of
```
Comment (zonque) [inline man/systemd-cgtop.xml]: Typo here: `pare`. Did you run `make man` locally?

**Comment by zonque** [comment]
Hmm, care to address the feedback @teg gave as well? As you force-pushed, we can't see them anymore here, but you should probably still have the email notifications ...

**Comment by utezduyar** [comment]
@zonque , I shouldn't have force pushed I guess until the very end. I think the ball is on @teg actually. @teg? 

screen shot 2015-06-17 at 8 50 56 am

**Comment by poettering** [comment]
Please merge this into a single commit with a good commit msg, these are two seperate patches now and the second one has a weird commit msg...

**Comment by utezduyar** [comment]
That is how I started working. I just stack patches until we all agree and then finally force-upload the branch. Otherwise, we are loosing inline comments on the patches.

**Pull request merged**